src/Symfony/Component/Process/Process.php

```diff
@@ -1220,6 +1223,36 @@ public function getStartTime(): float
         return $this->starttime;
     }
 
+    /**
+     * setWinOptions sets windows proc_open specific options
+     * useful with the option create_new_console to open a process and an detach it
+     * from your php script so it can run in background
+     * requires minimum PHP 7.4.4.
+     */
+    public function setWinOptions(array $options)
+    {
+        if ('\\' !== \DIRECTORY_SEPARATOR) {
+            throw new LogicException('setWinOptions is only for setting Windows proc_open options.');
+        }
+
+        $defaultOptions = $this->options;
+        $existingWinOptions = ['suppress_errors', 'bypass_shell', 'blocking_pipes', 'create_process_group', 'create_new_console'];
```

Comment:
Let's remove suppress_errors and bypass_shell, we don't want ppl to change them.

Reply:
OK, I removed them